app/views/boards/show.html.erb

```diff
@@ -0,0 +1,23 @@
+<div class="container mt-5">
+  <h2><%= @board.name %></h2>
+
+  <div class="container mt-4">
+    <div class="row row-cols-auto d-flex gap-4">
+      <% @lists.each do |list| %>
+        <div class="bg-light rounded" style="width: 18rem;">
+          <h4><%= list.name %></h4>
+          <ul class="list-group">
+            <li class="list-group-item">An item</li>
+            <li class="list-group-item">A second item</li>
+            <li class="list-group-item">A third item</li>
+            <li class="list-group-item">A fourth item</li>
+            <li class="list-group-item">And a fifth one</li>
+          </ul>
+        </div>
+      <% end %>
```

Comment:
Don't you think this could go in a partial? Abstract it.

Reply:
I've already done that on the current branch I'm working